<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#logoWidget{border-image: url(://Images/logo.png);}
#version{font-family: &quot;Microsoft Yahei&quot;;font-size:12px;color:rgb(254,254,254);}
#statusLabel{font:20px;color:#808080;}</string>
   </property>
   <widget class="QWidget" name="logoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>45</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="buttonZonesWidget" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="settingBtn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 0;</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/Images/setting.png</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="shutdownBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 0;</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/Images/shutdown_btn.png</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2" alignment="Qt::AlignRight">
         <widget class="QLabel" name="softVersionLab">
          <property name="text">
           <string>V1.0.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>45</y>
      <width>1222</width>
      <height>579</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="showGridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QWidget" name="listWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>270</height>
        </size>
       </property>
       <widget class="QTableWidget" name="RecordTableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>360</width>
          <height>250</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>250</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt &quot;Simplified Arabic&quot;;
text-align: center; </string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="rowCount">
         <number>10</number>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>66</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>22</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>姓名</string>
         </property>
         <property name="font">
          <font>
           <family>Simplified Arabic</family>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>比对时间</string>
         </property>
         <property name="font">
          <font>
           <family>Simplified Arabic</family>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>比对分值</string>
         </property>
         <property name="font">
          <font>
           <family>Simplified Arabic</family>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>比对结果</string>
         </property>
         <property name="font">
          <font>
           <family>Simplified Arabic</family>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>有效期</string>
         </property>
         <property name="font">
          <font>
           <family>Simplified Arabic</family>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="font">
          <font>
           <family>Simplified Arabic</family>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="ImageWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>285</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>-3</x>
          <y>-3</y>
          <width>431</width>
          <height>286</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="IDHeadLab">
           <property name="minimumSize">
            <size>
             <width>183</width>
             <height>47</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#IDHeadLab{border-image: url(:/Images/idImage.png);}
#version{font-family: &quot;Microsoft Yahei&quot;;font-size:12px;color:rgb(254,254,254);}
#statusLabel{font:20px;color:#808080;}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="livePushBtn">
           <property name="minimumSize">
            <size>
             <width>183</width>
             <height>47</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 0</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/Images/currImage.png</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>183</width>
             <height>47</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="IDImageLab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>183</width>
             <height>227</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">#IDImageLab{border-image: url(:/Images/default.png);}
#version{font-family: &quot;Microsoft Yahei&quot;;font-size:12px;color:rgb(254,254,254);}
#statusLabel{font:20px;color:#808080;}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="LiveImageBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>213</width>
             <height>227</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 0</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/Images/default.png</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>183</width>
             <height>227</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="VideoWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>285</height>
        </size>
       </property>
       <widget class="QLabel" name="VideoFrame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>380</width>
          <height>285</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="InfoWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>270</height>
        </size>
       </property>
       <widget class="QLabel" name="label_null0">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>250</width>
          <height>200</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="idNameLab">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>400</width>
          <height>35</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;Simplified Arabic&quot;;</string>
        </property>
        <property name="text">
         <string>姓      名:</string>
        </property>
       </widget>
       <widget class="QLabel" name="matchScoreLab">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>250</width>
          <height>35</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;Simplified Arabic&quot;;</string>
        </property>
        <property name="text">
         <string>比对分值:</string>
        </property>
       </widget>
       <widget class="QLabel" name="idNumberLab">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>400</width>
          <height>35</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;Simplified Arabic&quot;;</string>
        </property>
        <property name="text">
         <string>身份证号:</string>
        </property>
       </widget>
       <widget class="QLabel" name="idNationLab">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>45</y>
          <width>400</width>
          <height>35</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:18pt &quot;Simplified Arabic&quot;;</string>
        </property>
        <property name="text">
         <string>民      族:</string>
        </property>
       </widget>
       <widget class="QLabel" name="idExpireEndLab">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>115</y>
          <width>400</width>
          <height>35</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;Simplified Arabic&quot;;</string>
        </property>
        <property name="text">
         <string>有效期限:</string>
        </property>
       </widget>
       <widget class="QLabel" name="resultImgLab">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>140</y>
          <width>120</width>
          <height>120</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
